pkg/virtctl/vm/vm.go

```diff
@@ -285,6 +323,20 @@ func usage(cmd string) string {
 	return usage
 }
 
+func usageMemoryDump() string {
+	usage := `  #Dump memory of a virtual machine instance called 'myvm' to an pvc called 'memoryvolume'.
+  {{ProgramName}} memory-dump myvm --claim-name=memoryvolume
```

Comment:
just a thought; do we want this to be more consistent with hotplug virtctl cmds?
`{{ProgramName}} addvolume fedora-dv --volume-name=example-dv`
(make the PVC name positional arg?)

Reply:
It is the same, fedore-dv is the vm name same as in memorydump, and the pvc/volume-name is a required flag